bridges/modules/beefy/Cargo.toml

```
@@ -0,0 +1,62 @@
+[package]
+name = "pallet-bridge-beefy"
+version = "0.1.0"
+description = "Module implementing BEEFY on-chain light client used for bridging consensus of substrate-based chains."
+authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2021"
+license = "GPL-3.0-or-later WITH Classpath-exception-2.0"
+publish = false
+
+[lints]
+workspace = true
+
+[dependencies]
+codec = { package = "parity-scale-codec", version = "3.1.5", default-features = false }
```

we should align deps with the rest of the polkadot-sdk, e.g. to use latest:
```suggestion
codec = { package = "parity-scale-codec", version = "3.6.1", default-features = false }
```
or stick to the [workspace deps](https://github.com/paritytech/polkadot-sdk/blob/master/Cargo.toml#L547) or [workspace.dev.deps](https://github.com/paritytech/polkadot-sdk/blob/master/Cargo.toml#L591)

sure, we can do it as follow up, but also it will be nice here

---

Good point. Partially addressed here: https://github.com/paritytech/parity-bridges-common/pull/2910

But there are also some dependencies that have older versions on polkadot-sdk. For those I will open a separate PR in polkadot-sdk to update them.

---

The PR for upgrading the dependencies in `polkadot-sdk`: https://github.com/paritytech/polkadot-sdk/pull/3937